File: junk/python/fizzBuzz.py
def fizzBuzz():
	for x in range(0, 101):
		if x % 5 == 0 and x % 3 == 0:
			print("Fizzbuzz")
		elif x % 3 == 0:
			print("Fizz")
		elif x % 5 == 0:
			print("Buzz")
		else:
			print(x)

fizzBuzz()

File: junk/java/fizzBuzz.java
public class fizzBuzz{
	public static void main(String[] args){
		for(int i = 1; i <= 100; i++){
			StringBuilder fb = new StringBuilder();
			if(i % 15 == 0){
				fb.append("FizzBuzz");
			}
			else if(i % 3 == 0){ //because elegance is just out of my grasp
				fb.append("Fizz");
			}
			else if(i % 5 == 0){
				fb.append("Buzz");
			}
			else{
				System.out.println(i);
			}
			System.out.println(fb);
			System.out.println("\n");
		}
	}
}
